go.mod

```diff
@@ -571,23 +571,45 @@ require (
 	github.com/godror/godror v0.37.0
 	github.com/jmoiron/sqlx v1.3.5
 	github.com/kr/pretty v0.3.1
+	github.com/pulumi/pulumi/sdk/v3 v3.72.2
```

Comment:
Was this intentional? I don't see any new code in the agent using the pulumi SDK.